grakn-test/src/test/java/ai/grakn/test/graql/query/InsertQueryTest.java

```diff
@@ -417,7 +416,6 @@ public void testInsertReferenceByName() {
         //noinspection OptionalGetWithoutIsPresent
         EntityType newType = typeQuery.get("n").findFirst().get().asEntityType();
 
-        assertTrue(newType.asEntityType().isAbstract());
```

Comment:
Yes the test was attemping to add instances to an abstract type. So I stopped the type from being abstract which meant this line had to die.